compass/Coordinator.java

```diff
@@ -227,34 +231,20 @@ private void setup() throws ArgumentException {
   }
 
   private void start() throws ArgumentException, InterruptedException {
-    int bootstrap = config.bootstrap ? 0 : 3;
+    int bootstrapStage = 0;
```

Comment:
maybe, I am missing something, but if `config.bootstrap` is false, shouldn't this be 3?

Reply:
Read the full refactoring, the `bootstrapStage` is only used when bootstrap is true.